services/queue/src/api.js

```diff
@@ -93,6 +93,7 @@ let builder = new APIBuilder({
   context: [
     'db', // Database instance
     'taskGroupExpiresExtension', // Time delay before expiring a task-group
+    'signPublicArtifactURLs', // Whether to use AWS signed URLs for public artifacts
```

Comment:
for better or worse it seems like our standard in js is `Urls` rather than `URLs` so let's stick with that here.